src/azure-cli/azure/cli/command_modules/resource/_bicep.py

```diff
@@ -143,17 +143,17 @@ def is_bicep_file(file_path):
 
 def get_bicep_available_release_tags():
     try:
-        ca_file = certifi.where()
-        response = requests.get("https://aka.ms/BicepReleases", verify=ca_file)
+        os.environ.setdefault("CURL_CA_BUNDLE", certifi.where())
+        response = requests.get("https://aka.ms/BicepReleases")
```

Comment:
Per https://requests.readthedocs.io/en/latest/user/advanced/#ssl-cert-verification, there is no need to set `certifi.where()` to `CURL_CA_BUNDLE` at all. 

`requests` uses following CA bundles:
1. `REQUESTS_CA_BUNDLE`
2. `CURL_CA_BUNDLE`
3. `certifi.where()`

If `REQUESTS_CA_BUNDLE` or `CURL_CA_BUNDLE` is not set, `request` by default uses `certifi.where()`.